app/src/main/java/com/sci/coffeeandroid/feature/menudetails/data/datasource/CoffeeDetailRemoteDataSourceImpl.kt

```diff
@@ -0,0 +1,57 @@
+package com.sci.coffeeandroid.feature.menudetails.data.datasource
+
+import com.sci.coffeeandroid.feature.menudetails.domain.model.CoffeeModel
+import io.ktor.client.HttpClient
+import io.ktor.client.call.body
+import io.ktor.client.request.get
+
+class CoffeeDetailRemoteDataSourceImpl() : CoffeeDetailRemoteDataSource {
```

Comment:
FYI , plz don't touch any code if not ready yet for real remote datasource class , it can make forget you to delete fake data later .